.coveragerc

```diff
@@ -455,8 +455,12 @@ omit =
     homeassistant/components/nederlandse_spoorwegen/sensor.py
     homeassistant/components/nello/lock.py
     homeassistant/components/nest/*
-    homeassistant/components/netatmo/*
-    homeassistant/components/netatmo_public/sensor.py
+    homeassistant/components/netatmo/__init__.py
+    homeassistant/components/netatmo/binary_sensor.py
+    homeassistant/components/netatmo/api.py
+    homeassistant/components/netatmo/camera.py
```

Comment:
The sensor platform needs to be excluded too.

Reply:
duh... `homeassistant/components/netdata/sensor.py` just looked too similar right below. ;)